app/assets/javascripts/application.js

```diff
@@ -57,12 +57,17 @@ window.QPixel = {
 };
 
 $(document).on('ready', function() {
-  $("a.flag-link").bind("click", function(ev) {
+  $("a.flag-dialog-link").bind("click", function(ev) {
```

Comment:
I don't think this is possible, because arrow functions are scoped differently. Changing this results in to all cases of `this` and therefore also `self` to fail.

Reply:
Change `this` to `ev.target` - i.e. `const self = $(ev.target)`

Reply:
Tried that; didn't work with notifications. Given that the don't use it anymore anyway, I could simply drop it from there, which should allow making this change. Does that look okay?

Reply:
Which part are you referring to dropping? In general, you should be able to work most things off `ev.target`, just a matter of some HTML traversal until you find what you want.

Reply:
This:

https://github.com/codidact/qpixel/pull/21/files/627795dcb068cc83f9db9a35813082f9bfd151ad#diff-a9c3bd311eab80c9ebe6a69830f9ad02R26 (because it isn't used afterwards anyway)

Reply:
Link doesn't work for me, but if it's not used afterwards that should be okay to drop.